connection.c

```diff
@@ -62,6 +62,10 @@ Connection *ConnCreate(RedisRaftCtx *rr, void *privdata, ConnectionCallbackFunc
     conn->idle_callback = idle_cb;
     conn->free_callback = free_cb;
     conn->id = ++id;
+    if (rr->config->connection_timeout > 0) {
+        conn->timeout = RedisModule_Calloc(1, sizeof(struct timeval));
+        conn->timeout->tv_usec = rr->config->connection_timeout;
+    }
```

Comment:
@sjpotter I think we can always assume there's a non-zero connection timeout (btw the configuration code already does that), keep the `struct timeval` part of the connection and always enforce it.

Reply:
ok, removing the if